experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -309,9 +308,7 @@ function get_field(atmos_sim::ClimaAtmosSimulation, ::Val{:F_radiative_TOA})
     # save radiation source
     if radiation != nothing
         face_space = axes(atmos_sim.integrator.u.f)
-        z = parent(Fields.coordinate_field(face_space).z)
-        Δz_top = round(FT(0.5) * (z[end, 1, 1, 1, 1] - z[end - 1, 1, 1, 1, 1]))
-        n_faces = length(z[:, 1, 1, 1, 1])
+        n_faces = length(face_space.grid.vertical_grid.topology.mesh.faces)
```

Comment:
I think that there are better accessors. You might want to use `Spaces.vertical_topology(face_space)`

Reply:
Could we also rename this to `nz_faces`?